connectivity/connectivity-demos/src_hd/com/axonivy/connectivity/rest/ChatClient/ChatClientProcess.mod

```diff
@@ -52,38 +52,27 @@ Cs0 f5 109 160 211 160 #arcP
 Cs0 f6 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <elementInfo>
     <language>
-        <name>Client Side in JavaScript
+        <name>**Client Side in JavaScript**
 
 Not much is going on in here:
 - within this demo no server side processing is done with JSF
 - data is requested and shown via JavaScript directly within the client browser
-- see resources/chat.js and the ChatClient View for details</name>
-        <nameStyle>25,0,8
-180,7
-17,1,7
-9,7
-15,1,7
-12,7
-</nameStyle>
+- see *resources/chat.js* and the *ChatClient View* for details</name>
     </language>
```

Comment:
some evidence that Markdown is much easier to comprehend with a plain text viewer, than the old numeric nameStyle array